src/components/RIVTrafficLightsComponent.js

```diff
@@ -1,124 +1,226 @@
-import { CardContent, Typography, Grid, Box } from "@mui/material";
+import {
+  CardContent,
+  Typography,
+  Grid,
+  Box,
+  Button,
+  Tooltip,
+  IconButton,
+} from "@mui/material";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import ClickAwayListener from "@mui/material/ClickAwayListener";
 import RIVTrafficLightContext from "contexts/RIVTrafficLightContext";
+import { useState } from "react";
 import { useContext } from "react";
+import { AiOutlineInfoCircle } from "react-icons/ai";
 
 export default function RIVTrafficLightsComponent() {
   const { RIVTrafficLight, setRIVTraffiLight } = useContext(
     RIVTrafficLightContext
   );
+  const [tempRIVTrafficLight, setTempRIVTrafficLight] =
+    useState(RIVTrafficLight);
+
+  const createTooltipFunctions = () => {
+    const [open, setOpen] = useState(false);
+    const handleTooltipClose = () => {
+      setOpen(false);
+    };
+    const handleTooltipOpen = () => {
+      setOpen(true);
+    };
+
+    return {
+      open,
+      handleTooltipClose,
+      handleTooltipOpen,
+    };
+  };
+
+  const tooltips = {
+    sbend: createTooltipFunctions(),
+    depth: createTooltipFunctions(),
+    edge: createTooltipFunctions(),
+    wind: createTooltipFunctions(),
+  };
 
   return (
     <CardContent>
       <Typography
-        style={{ fontSize: 16, fontWeight: 550 }}
+        component="span"
+        style={{
+          fontSize: 16,
+          fontWeight: 550,
+          verticalAlign: "middle",
+        }}
         color="textSecondary"
         gutterBottom
       >
         Riskiarvojen esitysvärien raja-arvot:
-      </Typography>
-      <Grid container spacing={1}>
-        <Grid item xs={2.5}>
-          <Box
-            sx={{
-              bgcolor: "green",
-              color: "success.contrastText",
-              p: 1,
-            }}
-          >
-            Vihreä
-          </Box>
-        </Grid>
-        <Grid item xs={9.5}>
-          <input
-            type="float"
-            required
-            defaultValue={0}
-            style={{
-              width: 100,
-            }}
-          />
-          <label>{"≤ RIV <"}</label>
-          <input
-            type="float"
-            required
-            value={RIVTrafficLight.green}
-            onChange={(ev) =>
-              setRIVTraffiLight({
-                ...RIVTrafficLight,
-                green: Number(ev.target.value),
-              })
+        <ClickAwayListener onClickAway={tooltips.edge.handleTooltipClose}>
```

Comment:
tooltips.edge is used in "Väylän reuna ja reunan painokerroin" new tooltip for trafficlight could be done. Otherwise when here is only one tooltip in this component createTooltipFunctions is not necessary and these consts can be used as they are: 
const [open, setOpen] = useState(false);
    const handleTooltipClose = () => {
      setOpen(false);
    };
    const handleTooltipOpen = () => {
      setOpen(true);
    };